src/main/java/di/ConfigModule.java

```diff
@@ -6,12 +6,15 @@
 import com.movie_collection.bll.services.MovieService;
 import com.movie_collection.bll.services.interfaces.ICategoryService;
 import com.movie_collection.bll.services.interfaces.IMovieService;
+import com.movie_collection.dal.dao.MovieDAO;
+import com.movie_collection.dal.interfaces.IMovieDAO;
 import com.movie_collection.gui.controllers.controllerFactory.IControllerFactory;
 import com.movie_collection.gui.controllers.controllerFactory.ControllerFactory;
 
 public class ConfigModule extends AbstractModule {
     @Override
     public void configure(){
+        bind(IMovieDAO.class).to(MovieDAO.class);
```

Comment:
am I blind or we missing some documentation here ?

Reply:
https://github.com/TomassSimko/Private-Movie-Collection/pull/9/commits/ef7263ec26617d666f223e684bd1a8de326bd19e